### PR TITLE
[SPARK-50506][DOCS] Codify Spark Standalone documentation consistently

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
### What changes were proposed in this pull request?

This PR aims to codify `Spark Standalone` documentation more consistently.

### Why are the changes needed?

- https://spark.apache.org/docs/4.0.0-preview2/spark-standalone.html

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Manual review.

### Was this patch authored or co-authored using generative AI tooling?

No.

### CODE REVIEW

File: core/src/main/scala/org/apache/spark/internal/config/Deploy.scala [hunk 1]
```diff
@@ -45,7 +45,7 @@ private[spark] object Deploy {
 
   val RECOVERY_TIMEOUT = ConfigBuilder("spark.deploy.recoveryTimeout")
     .doc("Configures the timeout for recovery process. The default value is the same " +
-      "with ${WORKER_TIMEOUT.key}.")
+      s"with ${WORKER_TIMEOUT.key}.")
```
Comment (dongjoon-hyun) [inline core/src/main/scala/org/apache/spark/internal/config/Deploy.scala:48]: This is a configuration doc.

File: docs/spark-standalone.md [hunk 2]
```diff
@@ -611,37 +611,37 @@ via <code>http://[host:port]/[version]/submissions/[action]</code> where
 <code>action</code> is one of the following supported actions.
 
 <table class="spark-config">
-  <thead><tr><th>Command</th><th>Description</th><th>HTTP METHOD</th><th>Since Version</th></tr></thead>
+  <thead><tr><th>Command</th><th>HTTP METHOD</th><th>Description</th><th>Since Version</th></tr></thead>
```
Comment (dongjoon-hyun) [inline docs/spark-standalone.md:614]: This is a column switch to adjust widths.

**Review by huaxingao — APPROVED** [review]
LGTM. Thanks for the PR @dongjoon-hyun

**Comment by dongjoon-hyun** [comment]
Thank you so much, @huaxingao !

**Review by HyukjinKwon — APPROVED** [review]

**Comment by dongjoon-hyun** [comment]
Thank you, @HyukjinKwon .

**Comment by dongjoon-hyun** [comment]
Merged to master for Apache Spark 4.0.0 on February 2025.

**Pull request closed without merging**